{{cookiecutter.project_name}}/environments/kolla/configuration.yml

```diff
@@ -42,6 +42,21 @@ designate_ns_record: openstack.{{cookiecutter.domain}}
 
 {%- if cookiecutter.with_ceph|int %}
 
+##########################################################
+# ceph rgw integration
+
+enable_ceph_rgw: true
+enable_ceph_rgw_keystone: true
+
+ceph_rgw_swift_compatibility: false
+ceph_rgw_swift_account_in_url: true
+
+# All Ceph RGW hosts must be listed here
+ceph_rgw_hosts:
```

Comment:
We have three controller nodes with a specific Kolla "radosgw_interface" set as host variable. Can this be implemented as loop for group "ceph-control" and getting the ip addres from facts?

Reply:
We can do this in this way in the Ansible defaults. Will you open a separate issue for this? We also have the same in environments/infrastructure for the Cephclient service. Then we can also improve it there directly.